packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -224,6 +215,8 @@ If you're using [custom roles](https://help.okta.com/okta_help.htm?type=oie&id=e
 > **Note:** You can bind resource sets to many different types of resources, such as user groups, flows, authorization servers, and so on. You don't necessarily have to use the All Groups resource set in this use case. You can specify permissions for a specific subset of users in a custom role. See [Custom role assignment](/docs/concepts/role-assignment/#custom-role-assignment) and the [Custom Role assignment API operations](/docs/reference/api/roles/#custom-role-assignment-operations).
```

Comment:
Not entirely sure what are all the apis that are used for Org2Org so I cannot confirm if those recommended custom roles/resource sets are enough. The #oin-integration-platform team would know better.